src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -476,14 +498,25 @@ void ValidateContent(string expectedContent, string actualContent, string detail
                     {
                         // Post echos back the requested content, first buffering it all server-side, then sending it all back.
                         var s = new MemoryStream();
-                        await context.Request.Body.CopyToAsync(s);
+                        var buffer = new byte[serverBufferSize];
```

Comment:
Given that the server and client will be split to potentially separate processes, the serverBufferSize value should be determined by server logic alone.